pyvo/registry/regtap.py

```diff
@@ -824,8 +835,13 @@ def describe(self, verbose=False, width=78, file=None):
                 file=file)
 
         if verbose:
+            print("\n More info:")
             if self.reference_url:
-                print("More info: " + self.reference_url, file=file)
+                print("Reference url: " + self.reference_url, file=file)
```

Comment:
I'm not sure whether I'm too happy about writing "Reference URL" instead of "More info".  I guess the latter gives users a better idea what *should* be at the end of the reference (disregarding for the moment that sometimes operators don't to a good job with their reference URLs).  But if you're sure "Reference URL" is better, I'll budge.

Reply:
Makes sense, I reverted this too